paper/src/main/java/com/aleksey/castlegates/plugins/jukealert/JukeAlertManager.java

```diff
@@ -27,6 +27,9 @@ public boolean hasJukeAlertAccess(Location location, String groupName) {
 						)
 				{
 					LeverToggleAppender toggleAppender = snitch.getAppender(LeverToggleAppender.class);
+					if (toggleAppender == null) {
+						return false;

```

Comment:
I propose to use here `continue`
If there is more than one neighboring snitch then any of them should have possibility to trigger CG.
For example, **snitch 1** may not have LeverToggleAppender but **snitch 2** can have it and therefore we should trigger CG